src/modules/components/AlbumCard/AlbumCard.tsx

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { Box, Card, Typography, CardHeader, IconButton, CardMedia } from '@mui/material';
+import { useEffect } from 'react';
+import MoreVertIcon from '@mui/icons-material/MoreVert';
+import CropOriginalOutlinedIcon from '@mui/icons-material/CropOriginalOutlined';
+import { useAppDispatch, useAppSelector } from 'shared/hooks/redux-hooks';
+import { retrievePhotos } from 'shared/store/Photos/photoSlice';
+import { colors } from 'styles/variables';
+
+const AlbumCard = ({ title, idPhoto, countPhotos }: { title: string; idPhoto: number; countPhotos: number }): JSX.Element => {
+  const dispatch = useAppDispatch();
+  const photos = useAppSelector((store) => store.photo);
+
+  useEffect(() => {
+    dispatch(retrievePhotos([idPhoto]));
+  }, [dispatch]);
+
+  return (
+    <Card
+      sx={{
+        width: 280,
+        maxHeight: 370,
+        p: 0,
+
+        backgroundColor: 'transparent',
+        boxShadow: 'none',
+        position: 'relative',
+        '&:hover': {
+          '& .MuiIconButton-root': {
+            display: 'inline-flex',
+          },
+        },
+      }}
+    >
+      <CardHeader
+        sx={{
+          position: 'absolute',
+          right: 0,
+        }}
+        action={
+          <IconButton
```

Comment:
Is this icon really in the spec already?

Reply:
No, this case doesn't exist in spec but it takes no a lot of time